src/organizations/OrganizationEntity.js

```diff
@@ -1,11 +1,12 @@
+import { Rels, Classes, Actions } from '../hypermedia-constants';
+import { ActivityUsageEntity } from '../activities/ActivityUsageEntity.js';
 import { Entity } from '../es6/Entity.js';
 import { SimpleEntity } from '../es6/SimpleEntity.js';
-import { AlertsEntity } from './AlertsEntity.js';
-import { Rels, Classes, Actions } from '../hypermedia-constants';
+import { performSirenAction } from '../es6/SirenAction';
 import { NotificationCollectionEntity } from '../notifications/NotificationCollectionEntity';
 import { SequenceEntity } from '../sequences/SequenceEntity.js';
-import { ActivityUsageEntity } from '../activities/ActivityUsageEntity.js';
-import { performSirenAction } from '../es6/SirenAction';
+
+import { AlertsEntity } from './AlertsEntity.js';
```

Comment:
why this is reordered ?

Reply:
I don't recall touching this.

Reply:
The old one wasn't correct either. We generally alphabetize imports, with multi-imports appearing first. I added a commit to correct this.